onnxruntime/core/providers/webgpu/tensor/transpose.h

```diff
@@ -11,18 +11,21 @@
 namespace onnxruntime {
 namespace webgpu {
 
+constexpr static const uint32_t TILE_SIZE = 16;
```

Comment:
This may be better put in transpose.cc

Reply:
Move to onnxruntime::webgpu::Transpose::TILE_SIZE